### PR TITLE
Tickets/preops 1407

### PR DESCRIPTION
**Pull request by leannep**

### CODE REVIEW

**Comment by leannep** [comment]
@MelissaGraham It would be good if we could add a few lessons learnt into the Community Science section

File: RTN-041.tex [hunk 1]
```diff
@@ -36,17 +46,22 @@
...
 }
 
+
 % Change history defined here.
 % Order: oldest first.
 % Fields: VERSION, DATE, DESCRIPTION, OWNER NAME.
 % See LPM-51 for version number policy.
 \setDocChangeRecord{%
   \addtohist{1}{2022-08-02}{Unreleased.}{William O'Mullane}
+  \addtohist{2}{2023-04-05}{Added System Performance sections}{Leanne Guy}
```
Comment (womullan) [inline RTN-041.tex]: ISO dates  .. alos this should now problay be 1.0 and draft could be removed.
```suggestion 
  \addtohist{1.0}{2023-05-04}{Added System Performance sections}{Leanne Guy}
```

File: management.tex [hunk 2]
```diff
@@ -26,30 +24,52 @@ \subsection{Data Production}
...
 
 \begin{figure}
 \includegraphics[width=1.0\textwidth]{DpOrg}
 \caption{Data Production Leadership in operations - most of which was involved in DP0.2. \label{fig:DpOrg}}
 \end{figure}
 
+\subsection{System Performance}
+The Rubin Observatory System Performance department is responsible for ensuring that the LSST as a whole is proceeding with the efficiency and fidelity needed to achieve its science requirements at the end of the 10-year survey. 
+Aspects of this charge that were exercised extensively as part of DP0.2 were a) QA and performance characterization analyses of the LSST data products by the Verification and Validation team and b) enabling the community to access and analyze the data and publish results by the Community Science team.
+Members of the System Performance Leadership team (SPLT) who participated in Data Preview 0.2 was:
```
Comment (womullan) [inline management.tex]:
```suggestion
Members of the System Performance Leadership team (SPLT) who participated in Data Preview 0.2 were:
```

File: management.tex [hunk 3]
```diff
@@ -26,30 +24,52 @@ \subsection{Data Production}
...
+During the production of Data Preview 0.2 regular coordination meetings were held every two weeks (out of phase with the DPLT meeting) with minutes recorded on Confluence at \href{https://confluence.lsstcorp.org/display/LSSTOps/Data+Preview+Coordination+Meeting+Notes}{Data Preview Coordination Meeting Notes}.
+
+This meeting was attended by all the people directly involved in the data preview process: management, the processing infrastructure team, the science platform team, the execution team, the pipelines team, the verification and validation team, and the community science team.
 
 This allowed the different teams to report on status and bring up any issues that needed to be addressed and made everyone aware of progress.
-Data Preview 0.1 had been released during this period and this allowed us to also include feedback from the Community Engagement Team as they interacted with the existing delegates and prepared updated tutorials for DP0.2.
+Data Preview 0.1was released during this period and this allowed us to also include feedback from the Community Science Team as they interacted with the existing delegates and prepared updated tutorials for DP0.2.
```
Comment (womullan) [inline management.tex]:
```suggestion
Data Preview 0.1 was released during this period and this allowed us to also include feedback from the Community Science Team as they interacted with the existing delegates and prepared updated tutorials for DP0.2.
```

File: RTN-041.tex [hunk 4]
```diff
@@ -36,17 +46,22 @@
...
+The large and varied number of science analyses contributed by the DP0.2 delegates is testament to the success of Data Preview 0. 
 }
 
+
 % Change history defined here.
 % Order: oldest first.
 % Fields: VERSION, DATE, DESCRIPTION, OWNER NAME.
 % See LPM-51 for version number policy.
 \setDocChangeRecord{%
   \addtohist{1}{2022-08-02}{Unreleased.}{William O'Mullane}
```
Comment (womullan) [inline RTN-041.tex]: This was my mistake - the tag is 0.1 not 1.0
```suggestion
  \addtohist{0.1}{2022-08-02}{Unreleased.}{William O'Mullane}
```

File: RTN-041.tex [hunk 5]
```diff
@@ -1,7 +1,15 @@
 \documentclass[OPS,lsstdraft,authoryear,toc]{lsstdoc}
```
Comment (womullan) [inline RTN-041.tex]: This should be the issued version 1.0 I think .. tag 1.0 before merge
```suggestion
\documentclass[OPS,authoryear,toc]{lsstdoc}
```

**Review by womullan — CHANGES_REQUESTED** [review]
take out draft tag 1.0 after accepting/rejecting my few small edits ..

File: RTN-041.tex (hunk 1 above)
Comment (leannep) [reply RTN-041.tex]: It is ISO - just the wrong ISO date ..\

**Pull request merged**